mmdet/models/dense_heads/detr_head.py

```diff
@@ -132,29 +132,30 @@ def _init_layers(self) -> None:
             add_residual=False)
         # NOTE the activations of reg_branch here is the same as
         # those in transformer, but they are actually different
-        # in DAB DETR (prelu in transformer and relu in reg_branch)
+        # in DAB-DETR (prelu in transformer and relu in reg_branch)
         self.fc_reg = Linear(self.embed_dims, 4)
 
     # Note function _load_from_state_dict is deleted without
-    # supporting refactor-DETR in mmdetection2.0
+    # supporting refactor-DETR in mmdetection 2.x
```

Comment:
I can not remember. what does this mean.

Reply:
just remove it